utils/build/docker/nodejs/express4.Dockerfile

```diff
@@ -21,6 +21,8 @@ ENV PGDATABASE=system_tests
 ENV PGHOST=postgres
 ENV PGPORT=5433
 
+ENV DD_DATA_STREAMS_ENABLED=true
```

Comment:
shouldn't this be enabled depending on the scenario ? @cbeauchesne

Reply:
Indeed, good catch :) 

@ida613 , you need to add this var here : https://github.com/DataDog/system-tests/blob/main/utils/_context/_scenarios.py#L848

Reply:
The other languages are [doing the same](https://github.com/search?q=repo%3ADataDog%2Fsystem-tests%20DD_DATA_STREAMS_ENABLED&type=code). @cbeauchesne said we can keep it this way for now